src/contexts/AccountsContext.tsx

```diff
@@ -50,7 +54,7 @@ const AccountContextProvider = ({ children }: AccountContextProps) => {
   useEffect(() => {
     if (localStorageAccount) {
       const account = accounts.find(
-        (account) => account.address === localStorageAccount,
+        ({ address }: AccountAddressType) => address === localStorageAccount,
```

Comment:
`accounts` is of type `PolkadotAccount[]` so the type is inferred. You don't need to declare anything.
```suggestion
        ({ address }) => address === localStorageAccount,
```